code/datums/elements/cult_eyes.dm

```diff
@@ -0,0 +1,56 @@
+/**
+ * # Cult eyes element
+ *
+ * Applies and removes the glowing cult eyes
+ */
+/datum/element/cult_eyes
+
+/datum/element/cult_eyes/Attach(datum/target, override = FALSE)
+	. = ..()
+	if (!isliving(target))
+		return ELEMENT_INCOMPATIBLE
+
+	// Register signals for mob transformation to prevent premature halo removal
+	RegisterSignal(target, COMSIG_CHANGELING_TRANSFORM, .proc/set_eyes)
+	RegisterSignal(target, COMSIG_MONKEY_HUMANIZE, .proc/set_eyes)
+	RegisterSignal(target, COMSIG_HUMAN_MONKEYIZE, .proc/set_eyes)
+
+	if (override)
```

Comment:
This should instead be `initial_delay`, defaulting to `20 SECONDS`, but able to be set to zero.